tests/trilinos/sparse_matrix_add_03.cc

```diff
@@ -34,10 +34,10 @@
 void
 test()
 {
-  const auto MyPID{Utilities::MPI::this_mpi_process(MPI_COMM_WORLD)};
-  const auto NumProc{Utilities::MPI::n_mpi_processes(MPI_COMM_WORLD)};
+  const unsigned int MyPID   = Utilities::MPI::this_mpi_process(MPI_COMM_WORLD);
+  const unsigned int NumProc = Utilities::MPI::n_mpi_processes(MPI_COMM_WORLD);
```

Comment:
What a strange coincidence: the last three PRs I have looked at have involved brace initializers. Isn't the type on top, in C++11, `const std::initializer_list`? I am surprised that this ever worked in C++11 (not C++14).

Reply:
Yes, it is. That is exactly what the error message said.